packages/ruleset/src/functions/api-symmetry.js

```diff
@@ -310,7 +311,7 @@ function checkForGraphFragmentPattern(
           for (const [name, prop] of Object.entries(c.properties)) {
             logger.debug(`${ruleId}: checking canonical property '${name}'`);
 
-            const included = schemaHasLooseConstraint(
+            const included = schemaLooselyHasConstraint(
```

Comment:
I just moved the logic from this file to a new utility since we now need it for more than one rule. I also took the opportunity to change the name, since I wasn't a fan of the old one 😛 I'm open to other name suggestions as well!